rDNA/R/rDNA.R

```diff
@@ -1809,18 +1809,21 @@ print.dna_barplot <- function(x, trim = 30, attr = TRUE, ...) {
   print(x2)
 }
 
-#' Plot dna_barplot object.
+#' Plot \code{dna_barplot} object.
 #'
-#' Plot barplots generated from dna_barplot.
+#' Plot a barplot generated from \link{dna_barplot}.
```

Comment:
Function references need to do both link and code. See some other examples in the rDNA.R file where both are combined. Please change here and below.